hal/halx86/generic/x86bios.c

```diff
@@ -19,15 +19,17 @@
 /* This page serves as fallback for pages used by Mm */
 PFN_NUMBER x86BiosFallbackPfn;
 
-BOOLEAN x86BiosIsInitialized;
+BOOLEAN x86BiosInitialized;
```

Comment:
```suggestion
static BOOLEAN x86BiosInitialized;
```

Reply:
TBH, I find the switch to static more reasonable than the rename, which is just another change for the sake of changing it. Or do you want to save a few characters of source code to make Joachim happy?